server/static_src/scss/styles.scss

```diff
@@ -182,12 +182,16 @@ pre.scrolleable {
     display: block;
 }
 
-.zentral-logo.light {
-    background-image: url(../images/logo-light.svg);
+[data-bs-theme=light] {
+    .zentral-logo {
```

Comment:
Please do not use nested CSS, especially for one nested rule.

Reply:
This will grow, and this is `scss`, but as you wish.

Reply:
Yes, but nesting is still a young feature and we do not really need it since we already have a SCSS pipeline.